cax/cax.json

```diff
@@ -69,10 +71,15 @@
     "receive": "scp",
     "hostname": "tegner-login-1.pdc.kth.se",
     "directory": "/cfs/klemming/projects/xenon/xenon1t/data",
+    "dir_raw": "/cfs/klemming/projects/xenon/xenon1t/data",
+    "dir_root": "/cfs/klemming/projects/xenon/xenon1t/processed",
     "upload_options": [],
     "username": "bobau",
     "download_options": [
       "midway-login1"
+    ],
+    "pax_versions": [
+      "v4.10.0"
```

Comment:
i had originally implemented to handle multiple versions (in particular "head" and "v#.#.#") , and process.py took care of this by creating a directory for each. however, it seems we are now processing with only releases and not the head, so there's no particular reason to use multiple versions at once.